erigon-lib/kv/helpers.go

```diff
@@ -33,6 +33,24 @@ import (
 	"github.com/erigontech/erigon-lib/common"
 )
 
+// Adapts an RoDB to the RwDB interface (invoking write operations results in error)
+type RwWrapper struct {
+	RoDB
+}
+
+func (w RwWrapper) Update(ctx context.Context, f func(tx RwTx) error) error {
+	return errors.New("Update not implemented")
+}
+func (w RwWrapper) UpdateNosync(ctx context.Context, f func(tx RwTx) error) error {
+	return errors.New("UpdateNosync not implemented")
+}
+func (w RwWrapper) BeginRw(ctx context.Context) (RwTx, error) {
+	return nil, errors.New("BeginRw not implemented")
+}
+func (w RwWrapper) BeginRwNosync(ctx context.Context) (RwTx, error) {
+	return nil, errors.New("BeginRwNosync not implemented")
+}
```

Comment:
Does this mean that the support is not complete yet, so this PR is only setting the preliminary structure in place?